test/Microsoft.ML.Benchmarks/TextPredictionEngineCreation.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace micro
 {
-    [CoreJob]
+    [SimpleJob]
```

Comment:
what is the purpose of this attribute? So far it was set to run only on `.NET Core`. If we don't want to run it only for .NET Core, we can just remove the attribute.

Reply:
yes, as far as I know benchmark only runs on dotnet core

---
In reply to: [397732628](https://github.com/dotnet/machinelearning/pull/4968#discussion_r397732628) [](ancestors = 397732628)